pkg/handler/processor/guesser/type_ite6.go

```diff
@@ -40,6 +41,17 @@ func (_ *ite6TypeGuesser) GuessDocumentType(blob []byte, format processor.Format
 				return processor.DocumentITE6Generic
 			} else if strings.HasPrefix(statement.PredicateType, "https://in-toto.io/attestation/vuln/v0.1") {
 				return processor.DocumentITE6Vul
+			} else if strings.HasPrefix(statement.PredicateType, "https://in-toto.io/attestation/clearlydefined/v0.1") {
+				return processor.DocumentITE6ClearlyDefined
+			}
+			return processor.DocumentITE6Generic
+		}
+	}
+	var attV1Statement attestationv1.Statement
```

Comment:
this is separate now as the others are using the old attestation headers. Issue created: https://github.com/guacsec/guac/issues/2036